WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -1939,7 +2349,6 @@ static GSourceFuncs _handlerIntervention =
 
         string url = WKStringToString(urlStringRef);
 
-        browser->SetNavigationRef(navigation);
```

Comment:
keep it.

Reply:
Same as above, thanks, of course will keep it.